helpers/perfMonitor.js

```diff
@@ -0,0 +1,25 @@
+export class PerfMonitor {
+
+	constructor(host) {
+		this.constructedTime = window.performance.now();
+		this.host = host;
+	}
+
+	hostConnected() {
+		this.connectedTime = window.performance.now();
+	}
+
+	hostUpdated() {
+		if (!window.d2lPerfTestInProgress || this.renderedTime !== undefined) return;
+		setTimeout(async() => {
+			await this.host.updateComplete;
+			this.renderedTime = window.performance.now();
```

Comment:
Should this be `_renderedTime`?

Reply:
I just used non "private" variables in here since this isn't a mixin -- it's a controller so we don't need to worry about variable name collisions.